### PR TITLE
New header, from refills

### PR DESCRIPTION
**Pull request by cllns**
I added a new header, which I got from [refills.bourbon.io](http://refills.bourbon.io). It's pretty heavily modified, slightly for color, and more majorly restructured/reduced some of the SCSS to pass our scss-linter 😌. 

Also it's ~mobile-friendly~.

I added an icon/'logo' of a house from [material icons](https://www.google.com/design/icons/#ic_home). We'll likely want to change that, but it does the job for now, IMO.

Preview:
screen shot 2015-07-16 at 17 47 12

### CODE REVIEW

**Comment by nataliegroman** [comment]
I can see if I can mock up some new iconography. Refuge everything is up
for a revamp.

On Thu, Jul 16, 2015 at 2:50 PM, sean collins <notifications@github.com>
wrote:

> I added a new header, which I got from refills.bourbon.io. It's pretty
> heavily modified, slightly for color, and more majorly restructured/reduced
> some of the SCSS to pass our scss-linter 😌.
>
> Also it's ~mobile-friendly~.
>
> I added an icon/'logo' of a house from material icons
> <https://www.google.com/design/icons/#ic_home>. We'll likely want to
> change that, but it does the job for now, IMO.
>
> Preview:
> [image: screen shot 2015-07-16 at 17 47 12]
> <https://cloud.githubusercontent.com/assets/632942/8735972/d64153e0-2be2-11e5-8390-337d95c4cea2.png>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/RefugeRestrooms/refuge_housing/pull/11
> Commit Summary
>
>    - Add Header from Refills
>    - Clean up footer markup/css
>    - Add missing Header js
>
> File Changes
>
>    - *M* .scss_lint.yml
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-0>
>    (13)
>    - *A* app/assets/images/home.png
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-1>
>    (0)
>    - *A* app/assets/javascripts/header.js
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-2>
>    (13)
>    - *M* app/assets/stylesheets/application.scss
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-3>
>    (2)
>    - *M* app/assets/stylesheets/partials/_footer.scss
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-4>
>    (12)
>    - *D* app/assets/stylesheets/partials/_header.scss
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-5>
>    (40)
>    - *M* app/assets/stylesheets/partials/_layout.scss
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-6>
>    (3)
>    - *M* app/assets/stylesheets/partials/_posts.scss
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-7>
>    (2)
>    - *A* app/assets/stylesheets/refills/_header.scss
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-8>
>    (190)
>    - *M* app/views/application/_footer.html.erb
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-9>
>    (62)
>    - *M* app/views/application/_header.html.erb
>    <https://github.com/RefugeRestrooms/refuge_housing/pull/11/files#diff-10>
>    (73)
>
> Patch Links:
>
>    - https://github.com/RefugeRestrooms/refuge_housing/pull/11.patch
>    - https://github.com/RefugeRestrooms/refuge_housing/pull/11.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RefugeRestrooms/refuge_housing/pull/11>.
>



-- 
Natalie Roman
Collective Digital Studio - Network Development Designer
Beverly Hills, CA
765.713.9558
http://natalieroman.com

**Comment by cllns** [comment]
AWESOME! Please do @nataliegroman :) 👌  

Here were some ideas I had earlier:
messages image 445290639
messages image 717023781
messages image 1396482826

Also @mi-wood and I discussed possibly adding a circle around it, to make it more safe/secure/comforting.

But don't feel limited to that, it was just a whim I had.

**Comment by tkwidmer** [comment]
@cllns  I actually really like that.

File: app/views/application/_footer.html.erb [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
-  </a>
-  <a class="icon-link" href="http://blog.refugerestrooms.org/">
-    <img alt="Refuge's blog on tumblr" class="icon icon-tumblr" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <a class="icon-link" href="<%= contact_path %>">
-    <img alt="Email Us" class="icon icon-email" src="<%= asset_path("clear.png") %>"/>
-  </a>
-  <br/>
-  <%= "\&copy; copyleft #{Date.today.year} refuge housing.".html_safe %>
+  <div class="social-icons">
```
Comment (mi-wood) [inline app/views/application/_footer.html.erb]: Is there a way to provide alt text to these links for accessibility?

Comment (tkwidmer) [inline app/views/application/_footer.html.erb]: you can pass in any html parameter as data. so here you would just use 
```
alt: "text you want to put as alt.
```

Comment (tkwidmer) [inline app/views/application/_footer.html.erb]: http://stackoverflow.com/questions/8734722/rails-link-to-helper-with-data-attribute

Comment (tkwidmer) [inline app/views/application/_footer.html.erb]: you can put it in under a :data key, or I think you can just put them in and rails is smart enough.

**Comment by cllns** [comment]
I guess `alt` is just for images, and it's called `title` for links.

**Comment by tkwidmer** [comment]
you could also take this approach. link_to will take a block. So you can do a link_to with an img_tag inside of it.

http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#M001882

**Comment by cllns** [comment]
I refactored the `` tag out, so I'd rather not add it back in. Is there some accessibility reason why an image tag is better?

**Comment by tkwidmer** [comment]
http://webaim.org/articles/gonewild/#title according to this? it looks like the title attribute isn't ready by screen readers.

**Comment by tkwidmer** [comment]
looks like this is the most concice way to do a link_to with an image_tag

```
link_to image_tag("Search.png", :border => 0, :alt => '', :title => ''), pages_search_path, :class => 'dock-item'
```

**Pull request merged**